internal/ci/ci.go

```diff
@@ -81,3 +94,179 @@ func GetProvider() Provider {
 
 	return None
 }
+
+func getCI(provider Provider) CI {
+	var ci CI
+	if reflect.DeepEqual(provider, AppVeyor) {
+		ci = CI{
+			Provider:  provider,
+			OriginURL: os.Getenv("APPVEYOR_URL"),
+			Repo:      os.Getenv("APPVEYOR_REPO_NAME"),
+			RefName:   os.Getenv("APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH"),
+			SHA:       os.Getenv("APPVEYOR_REPO_COMMIT"),
+			User:      os.Getenv("APPVEYOR_REPO_COMMIT_AUTHOR"),
```

Comment:
In some cases you use the commit author, in other cases it's the user that kicked off the CI build. We should really stick to one of them. If we want to capture both, then let's introduce two fields. It's totally fine if a particular CI provider doesn't provide all the fields, in which case we leave those empty.

Reply:
Ok. Let me make it more clear.